gen/host.go

```diff
@@ -147,6 +147,19 @@ func genHost(g *protogen.GeneratedFile, f *fileInfo, service *serviceInfo) {
 				config:  config,
 			}, nil
 		}`)
+	g.P()
+
+	// Close plugin
+	g.P(fmt.Sprintf("func (p *%s) Close(ctx %s) (err error) {",
+		pluginName,
+		g.QualifiedGoIdent(contextPackage.Ident("Context")),
+	))
+	g.P(`if r := p.runtime; r != nil {
```

Comment:
let's merge these two g.P into single one for readability to see what's going on under Close function